fastlane/screenshots/jetpack-config.json

```diff
@@ -2,7 +2,7 @@
 	"version": 0.1,
 	"shadow_offset": 0,
 	"background_color": "#016087",
-	"stylesheet": "playstoreres/assets/style.css",
+	"stylesheet": "screenshots/assets/style.css",
 	"devices": [
 		{
 			"name": "Pixel 2 XL",
```

Comment:
Good point, since it's not used in any of the subsequent `entries` anyway…

PS: Hopefully, once [release-toolkit#429](https://github.com/wordpress-mobile/release-toolkit/pull/429) gets approved and land, we will also be able to later convert the `{app}-config.json` config file into a `{app}-config.yaml` config file, which will allow us to start using YAML anchors (e.g. use `name: &phone "Pixel 3"` under `devices` declaration, then just use `device: *phone` in the entries, without having to replace the device name in all the places if/when we decide to change the device models again… But we'll need [release-toolkit#429](https://github.com/wordpress-mobile/release-toolkit/pull/429 for that first.

Reply:
Addressed in 443598b8b991cc4a19c4002d38d3e4bd04161a76

Reply:
Nice, thank you for addressing this @AliSoftware ! 🙇 

> PS: Hopefully, once https://github.com/wordpress-mobile/release-toolkit/pull/429 gets approved and land, we will also be able to later convert the {app}-config.json config file into a {app}-config.yaml config file, which will allow us to start using YAML anchors (e.g. use name: &phone "Pixel 3" under devices declaration, then just use device: *phone in the entries, without having to replace the device name in all the places if/when we decide to change the device models again… But we'll need [release-toolkit#429](https://github.com/wordpress-mobile/release-toolkit/pull/429 for that first.

Coolio! 🤞